swaynag/config.c

```diff
@@ -113,20 +115,21 @@ int swaynag_parse_options(int argc, char **argv, struct swaynag *swaynag,
 		"  -v, --version                   Show the version number and quit.\n"
 		"\n"
 		"The following appearance options can also be given:\n"
-		"  --background RRGGBB[AA]         Background color.\n"
-		"  --border RRGGBB[AA]             Border color.\n"
-		"  --border-bottom RRGGBB[AA]      Bottom border color.\n"
-		"  --button-background RRGGBB[AA]  Button background color.\n"
-		"  --text RRGGBB[AA]               Text color.\n"
-		"  --button-text RRGGBB[AA]        Button text color.\n"
-		"  --border-bottom-size size       Thickness of the bar border.\n"
-		"  --message-padding padding       Padding for the message.\n"
-		"  --details-border-size size      Thickness for the details border.\n"
-		"  --button-border-size size       Thickness for the button border.\n"
-		"  --button-gap gap                Size of the gap between buttons\n"
-		"  --button-dismiss-gap gap        Size of the gap for dismiss button.\n"
-		"  --button-margin-right margin    Margin from dismiss button to edge.\n"
-		"  --button-padding padding        Padding for the button text.\n";
+		"  --background RRGGBB[AA]          Background color.\n"
+		"  --border RRGGBB[AA]              Border color.\n"
+		"  --border-bottom RRGGBB[AA]       Bottom border color.\n"
+		"  --button-background RRGGBB[AA]   Button background color.\n"
+		"  --text RRGGBB[AA]                Text color.\n"
+		"  --button-text RRGGBB[AA]         Button text color.\n"
+		"  --border-bottom-size size        Thickness of the bar border.\n"
+		"  --message-padding padding        Padding for the message.\n"
+		"  --details-border-size size		Thickness for the details border.\n"
```

Comment:
This should remain as spaces between the option and description

Reply:
It also looks like this doesn't need to be changed (the entire section that is). The added option is shorter than the rest.